cmd/lakefs/cmd/kv.go

```diff
@@ -361,6 +410,29 @@ Strategies:
 	},
 }
 
+// createRepoPartitionDump creates a dump of a specific repository's partition
+func createRepoPartitionDump(ctx context.Context, kvStore kv.Store, repoName string) (*kv.DumpFormat, error) {
+	// Get repository record from graveler partition
+	repoID := graveler.RepositoryID(repoName)
+	repoPath := graveler.RepoPath(repoID)
+
+	var repoData graveler.RepositoryData
+	_, err := kv.GetMsg(ctx, kvStore, graveler.RepositoriesPartition(), []byte(repoPath), &repoData)
+	if err != nil {
+		if errors.Is(err, kv.ErrNotFound) {
```

Comment:
Use just errors.Is , which is false for nil. This style is used throughout lakeFS.